lib/src/base-styles/_nx-globals.scss

```diff
@@ -47,3 +47,14 @@ $-nx-scrollable-default-height: 400px;
 .nx-blockquote {
   @extend %nx-blockquote;
 }
+
+//Class that allows making content visible only to screen readers but not visually
```

Comment:
I've never understood why online examples make this so complicated.  You really don't need to deal with clipping or even absolute positioning, you can just make the element 0x0 and then either make overflow hidden or make the text transparent.

Reply:
> I did a little bit of research on this and turns out it's not really recommended to set an element's width or height to 0px. Check out [this article](https://webaim.org/techniques/css/invisiblecontent/).

Huh, that's annoyingly inconsistent - the intent is the same whether the content is hidden via clipping or by making the element smaller, so it seems like it ought to be treated the same by screenreaders.  But let's stick with the clip approach then